src/framework/audio/internal/worker/sequenceplayer.cpp

```diff
@@ -37,18 +37,23 @@ SequencePlayer::SequencePlayer(IGetTracks* getTracks, IClockPtr clock, const mod
         seekAllTracks(m_clock->currentTime());
     });
 
-    m_clock->statusChanged().onReceive(this, [this](const PlaybackStatus status) {
-        audioEngine()->mixer()->setIsActive(status == PlaybackStatus::Running);
+    m_clock->countDownEnded().onNotify(this, [this]() {
+        audioEngine()->mixer()->setIsActive(m_clock->status() == PlaybackStatus::Running);
     });
```

Comment:
Aren't we losing basically `audioEngine()->mixer()->setIsActive(false);` when playback is stopped?